src/main/studio/kdb/Config.java

```diff
@@ -306,6 +306,33 @@ private void init(String filename, Properties properties) {
             }
         }
 
+        if (!Files.exists(file) && System.getProperty("os.name").startsWith("Windows")) {
+            System.out.println("Config not found in userprofile. Trying legacy path.");
+            //Old Java versions returned a different place for user.home on Windows.
```

Comment:
Out of curiosity - what java version returns Desktop for "user.home" ? I tried to Google, and found that someone experience sporadic problems - but the location was different.
https://coderanch.com/t/517573/java/System-getProperty-user-home-unreliable

May be instead of such code, user can copy the config to his/her user home?

BTW, also you can access native Windows API where I believe should be method to load from Registry. See WindowsAppUserMode class for an example to access Windows native functions.

Reply:
"user can copy the config to his/her user home"
That is the very motivation of this change, to NOT require the user to do anything. We were getting a lot of support requests about server lists disappearing after an upgrade, despite there being docs available about where the settings are and where they should be copied.